schema/fixtures/workflow/verbs/destructure/plain_object_remove_column_no_keys/expected.json

```diff
@@ -0,0 +1,66 @@
+{
+    "ID": [
```

Comment:
the columnar json is a little ugly, but that's more of an aside than anything

Reply:
Agreed, but it is the standard for pandas and built-in to arquero so it kept the file loading minimal